docs/api_docs/python.rst

```diff
@@ -0,0 +1,170 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _getting_started_python_api:
+
+******************************
+Getting Started with Python
+******************************
+
+Installation
+==================
+.. note::
+    We heavily recommend to manage your projects with some sort of virtual environment, for example Pipenv. For instructions to install Pipenv for Mac or Windows click `here <https://medium.com/@mahmudahsan/how-to-use-python-pipenv-in-mac-and-windows-1c6dc87b403e>`_. For installation instructions for Linux click `here <https://github.com/pypa/pipenv>`_.
+
+Installing with Pipenv
+-----------------------------
+In order to install the Vertiq python API into your pipenv virtual environment, simply run ``pipenv install iqmotion`` within your virtual environment.
+
+Installing with Pip
+-----------------------------
+In order to install the Vertiq python API via pip, simply run ``pip install iqmotion``.
```

Comment:
Throwing down a link to our PyPi for more information for people who like to look at that stuff may be good here: https://pypi.org/project/iqmotion/